core/src/main/resources/reference.conf

```diff
@@ -321,6 +321,19 @@ google {
 # They can be enabled individually in the engine.filesystems stanza and in the config.filesystems stanza of backends
 # There is a default built-in local filesystem that can also be referenced as "local" as well.
 filesystems {
+  blob {
+    class = "cromwell.filesystems.blob.BlobPathBuilderFactory"
+    # Use to share a unique global object across all instances of the factory
+    global {
+      # Class to instantiate and propogate to all factories
+      class = ""
+      config {
+        credential = ""
+        stores = "configuration"
+        endpoint = "https://coaexternalstorage.blob.core.windows.net/"
```

Comment:
I thought I remembered reading something about the `blob.core` segment varying under some conditions, but this page says it does not: https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview#standard-endpoints

However, it does include some information about a preview feature that lets you build a DNS zone into the endpoint. So probably still good to have the flexibility of providing the full endpoint, rather than the bare storage account name.

Reply:
So I think I am planning on removing the config for now, based on one of your other comments and since we talked about setting this up in a separate issue. I did an experiment to allow for a DNS field for now in addition to the storage account name, but I am not clear that we actually need it either, so I could walk that back, but it wasn't too bad to add as another configurable field